crates/rollup-boost/src/cli.rs

```diff
@@ -93,6 +93,15 @@ pub struct RollupBoostArgs {
     #[arg(long, env)]
     pub block_selection_policy: Option<BlockSelectionPolicy>,
 
+    /// Should we use the l2 client for computing state root
+    #[arg(long, env, default_value = "false")]
+    pub use_l2_client_for_state_root: bool,
+
+    /// Allow all engine API calls to builder even when marked as unhealthy
+    /// This is default true assuming no builder CL set up
+    #[arg(long, env, default_value = "true")]
```

Comment:
```suggestion
    #[arg(long, env, default_value = "false")]
```
default should meant to be false?

Reply:
updated